.gitignore

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
```

Comment:
Who creates these ctxt and mtj files? I don't have them.

Reply:
I don't know about these. Again, this is just the standard github .ignore file for the java language. It's a cooperative effort that can be found here:
https://github.com/github/gitignore